Topics/User_Experience/Mobile-First_Design.md

```diff
@@ -0,0 +1,54 @@
+
+## Benefits
+
+
+
+* Enhanced UX for mobile users
+    * As accessing web apps through phones has become the dominant mode of interaction — with more than [50%](https://www.statista.com/statistics/277125/share-of-website-traffic-coming-from-mobile-devices/#:~:text=Mobile%20accounts%20for%20approximately%20half,since%20the%20beginning%20of%202017.) of web traffic coming from mobile devices since 2022 — creating UIs with the mobile-first design approach ensures that the UX for mobile devices is not compromised from graceful degradation
+* Prioritized content
+    * With a smaller screen, the UI for mobile devices will likely have to cut out any unnecessary details or rearrange content, allowing your designs to be cleaner and more concise
+    * For instance, having a large block of text that spans your entire phone screen might lead you to shorten the text or break it up for an improved UX; the lengthiness of that body of text might have been overlooked initially if using the desktop-first approach
+* Improved accessibility
+    * Mobile-first design emphasizes legible typography and text sizes, ensuring readability and navigability for your web app
+* Reduced redesigning
+    * Adopting this approach can reduce the need for extensive redesign efforts later in the process if some critical features on the desktop version, such as hover effects, cannot be easily translated into the mobile version
+* Faster page loading times
+    * With a reduced amount of code, images, and resources required for the web app, losing speed can also be reduced
+
+## Drawbacks
+
+* Learning curve
+    * Most web app developers are accustomed to the desktop-first approach, thus shifting to this new strategy that requires a different way of thinking and planning requires some effort
+* Restricted design creativity and flexibility
+    * Mobile designs are more limited than desktop versions, which may put more restrictions on the content and features you can include
+    * These constraints stem from the limited screen size and reliance on touch interactions, as designing the mobile version requires meticulous consideration to arrange the content in a touch-friendly and legible manner; this involves ensuring the buttons are sized appropriately for touch, accommodating all text within the narrow screen, and also minimizing the need for excessive scrolling
+
+## General Steps
+
+1. Content inventory
+    * Plan out all the content and elements you need for your web pages
+2. Visual hierarchy
+    * Create a priority list of all the visuals you will include on the page, taking into consideration the size of images, colours, and typography, as these properties can guide the user’s eye to understand the relative significance of the elements
```

Comment:
Perhaps you can elaborate more on where _Step 1_ and _Step 2_ should be done? Should they also be done on "Figma, Canva, Google Drawings/Slides ..."? If so, I think its beneficial to include this bullet point (in quotes below) at the start of your _General Steps_ rather than later in _Step 3_ as it helps the reader understand _where_ they should be "plan[ing] out all the content" or "creat[ing] a priority list of all the visuals".

> These designs can be made on Figma, Canva, Google Drawings/Slides, or whatever design tool that will help you plan out the exact format of your page

Reply:
Great suggestion! I'll be sure to elaborate more on those steps.

Reply:
The link is great!